test/core/list/sort.wren

```diff
@@ -0,0 +1,10 @@
+var list = [4, 1, 3, 2].sort()
+System.print([4, 1, 3, 2].sort()) // expect: [1, 2, 3, 4]
```

Comment:
Does this test pass? I don't see `List.sort()` returning any value, so I would expect this to print `null`. Did you mean to `return this` at the end of `sort`?

Reply:
With this [issue](https://github.com/wren-lang/wren/issues/801) open the tests don't pass.

Reply:
@bjorn one liner body/method/function does implicitly return.

Reply:
@mhermier the tests were returning null

Reply:
@mhermier I know, but since the `sort(_)` function was returning `null`, so did the one-line version of `sort()`.

Reply:
So `sort(_)` should return `this` and the one-liner `sort()` should be reverted back to a stub function?